simulator/mips/mips_instr.h

```diff
@@ -326,12 +326,14 @@ class FuncInstr
         uint32 lo = NO_VAL32;
 
         FuncInstr() = delete; // constructor w/o arguments for ports
+ 
+        FuncInstr( const FuncInstr& instr) = default;
 
-        explicit
+        explicit   
         FuncInstr( uint32 bytes, Addr PC = 0,
                    bool predicted_taken = false,
                    Addr predicted_target = 0);
-
+                   
```

Comment:
Please remove trailing spaces here and in the line above. Actually, try to keep your patch to contain as low changes as it is possible.